code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -358,32 +351,34 @@
 	else
 		switch(build_step)
 			if(0)
-				if(istype(W, /obj/item/healthanalyzer))
-					if(!user.unEquip(W))
+				if(istype(I, /obj/item/healthanalyzer))
+					if(!user.drop_item())
 						return
-					qdel(W)
+					qdel(I)
 					build_step++
 					to_chat(user, "<span class='notice'>You add the health sensor to [src].</span>")
 					name = "First aid/robot arm/health analyzer assembly"
-					overlays += image('icons/obj/aibots.dmi', "na_scanner")
+					update_icon()
 
 			if(1)
-				if(isprox(W))
-					if(!user.unEquip(W))
+				if(isprox(I))
+					if(!user.drop_item())
 						return
-					qdel(W)
+					qdel(I)
 					build_step++
 					to_chat(user, "<span class='notice'>You complete the Medibot. Beep boop!</span>")
 					var/turf/T = get_turf(src)
-					var/mob/living/simple_animal/bot/medbot/S = new /mob/living/simple_animal/bot/medbot(T)
-					S.skin = skin
-					S.name = created_name
-					S.bot_core.req_one_access = req_one_access
-					S.treatment_oxy = treatment_oxy
-					S.treatment_brute = treatment_brute
-					S.treatment_fire = treatment_fire
-					S.treatment_tox = treatment_tox
-					S.treatment_virus = treatment_virus
+					if(!syndicate_aligned)
```

Comment:
I'm not a fan of this, but the problem is, storing all the unique behaviors and vars to Syndicate medibots, on an assembly and medkit, isn't really worth it, not to mention, you'd have to have a ton more arguments for `medbot/New` for this to work; it's easier just to do this here, and far cleaner.